src/scrabble.js

```diff
@@ -1,5 +1,58 @@
-function scrabble() {
+function scrabble(scrabbleWord) {
+  if (!scrabbleWord) return 0
+  const libary = {
```

Comment:
Declaring the const library at the top would give more visibility to the function scrabble